lib/resque/cluster/config.rb

```diff
@@ -62,10 +70,78 @@ def log_errors
 
       private
 
+      def host_maximums
+        case config_type
+        when :separate
```

Comment:
@babelfish It seems to be that it would be easier to rip out if they were objects that each responded to `#config`